java/registry/src/main/java/dev/sunbirdrc/registry/service/FileStorageService.java

```diff
@@ -139,4 +128,32 @@ public ComponentHealthInfo getHealthInfo() {
 			return new ComponentHealthInfo(getServiceName(), false, CONNECTION_FAILURE, e.getMessage());
 		}
 	}
+	public DocumentsResponse updateFiles(MultipartFile[] files, String requestedURI) {
```

Comment:
this is part of another Pull Request
we don't need this here
changes of this complete file can be removed